### ISSUE TITLE
"Get help with this page." service link

### ISSUE THREAD

**Issue by shabana-ali** [hmrc/accessibility#30]
Multiple issues with this feature.:

- the link does not declare it as a collapsed/expanded section and so when this is clicked on the screen reader user is not alerted to the expanded form
- the form is not styled in accordance to the design system
- on error, there is no error summary
- HTML code errors are introduced in the form of multiple labels for each input field
- this feature does not work if JavaScript is disabled and the user is redirected to a page that is also inaccessible, e.g. no h1 on https://www.staging.tax.service.gov.uk/contact/problem_reports_nonjs?service=pensionannualallowancecalculatorfrontend

![screenshot-get-help-with-this-page](https://user-images.githubusercontent.com/59563139/84159111-99ae2600-aa64-11ea-973c-2abf2c4a0e12.png)

**Comment by adamliptrot-oc** [comment]
This is on the Plat-UI roadmap as part of the wider contact frontend work.
The intention is that the link will become a standard link and the form page will be reworked and made stand-alone.

**Comment by ashfaqhussain357** [comment]
Fixed
